test/functional/test_ivn.py

```diff
@@ -124,21 +138,18 @@ def start_node_with_config(self, node_name, ns_name, boot_image, iso):
                                                      "mac": "00:60:16:93:b9:1a"})
             fake_node["compute"]["networks"][0]["port_forward"] = [{"outside": 8022, "inside": 22, "protocal": "tcp"}]
             fake_node["compute"]["networks"][0]["mac"] = "00:60:16:93:b9:2d"
```

Comment:
recommend using a variable to represents the MAC address.

Reply:
not necessary , there will be four variables here if we named them

Reply:
if not using variable, line 38, 127, 130 and line 39, 135, 137 must keep same manually.